nomenclature/definitions/variable/technology/technologies.yaml

```diff
@@ -535,3 +535,106 @@ Investment|Energy Supply|Liquids|Oil:
                 capturing equipment should be included but not the one on
                 transport and storage.
    unit: billion US$2010/yr
+   
+   # description of residential demand flexibility potentials for 10 household technologies - potential curtailable load
+   
+Potential|Electricity|Load Shifting|Residential|Refrigeration:
+   description: Maximum curtailable load in a given region/time frame,
```

Comment:
it is not necessary to specify "in a given region/time frame", this should be obvious anyway by having a region and a subannual column - and it removes the clutter from the descriptions

Reply:
ok, will change now, thanks!